docs/design/upstream-datamover.md

```diff
@@ -66,7 +66,7 @@ The new velero `DataUpload` and `DataDownload` will be added.
 
 #### OADP container images
 
-References to `velero-plugin-for-vsm` and `volume-snapshot-mover` images need to be removed from `config/manager/manager.yaml`, golang code, and `deploy/disconnected-prep.sh`.
+References to `velero-plugin-for-vsm` and `volume-snapshot-mover` images need to be removed from `config/manager/manager.yaml`, golang code, and `scripts/disconnected-prep.sh`.
```

Comment:
Maybe would also need changes in prod docs

Reply:
maybe not? https://github.com/search?q=repo%3Aopenshift%2Fopenshift-docs+disconnected-prep&type=code Is there other places our docs live?